source/_docs/modules-plugins-known-issues.md

```diff
@@ -464,6 +464,16 @@ This workaround may potentially break again with the next plugin update, and you
 
 <hr>
 
+### [SmartCrawl Pro](https://premium.wpmudev.org/project/smartcrawl-wordpress-seo/){.external}
+**Issue**: The sitemap url link by the plugin produces a `500 Internal Server Error` on Test and Live environments which results to a PHP error: `class not found WP_Filesystem_Direct`. See more [details about the issue](https://premium.wpmudev.org/forums/topic/smartcrawl-pro-class-wp_filesystem_direct-not-found){.external}.
+
+**Solution**: The plugin is failing to implement a direct `FS_METHOD` in Test and Live environments resulting to that error. Adding `define('FS_METHOD', 'direct');` in the `wp-config.php` before the `/* That's all, stop editing! Happy Pressing. */` would fix the issue.
+
+Alternative plugins that have an xml sitemap feature that works well in the platform are [Google Sitemap Generator](https://wordpress.org/plugins/google-sitemap-generator/){.external} or [Yoast](https://wordpress.org/plugins/wordpress-seo/){.external}. 
+
+[Upgrade your site's PHP version](/docs/php-versions) to 5.5, 5.6, or 7.0.
```

Comment:
```suggestion
```

Not sure why this is here, unless there'a another issue with the plugin on outdated PHP versions.

Reply:
Sorry about that, I think I accidentally pasted some content elsewhere along with this PR so that is not included